index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+ <html lang="en">
+    <head>
+       <title>The Code Magazine</title>
+       <meta http-equiv="Content-Type" content="text/html;charset=UTF-8">
+    </head>
+    <body>
+        <h1>📘The Code Magazine</h1>
+        <h2>The Basic Language of the Web: HTML</h2>
+        <img src="C:/As-gard/Tekwill/Tekwill-Andrian/img/laura-jones.jpg" alt="laura-jones">
```

Comment:
Since you have the images in your project use relative path to the images instead of absolute